titiler/pgstac/backend.py

```diff
@@ -271,7 +271,10 @@ def _reader(
             with self.reader(item, tms=self.tms, **self.reader_options) as src_dst:
                 return src_dst.tile(x, y, z, **kwargs)
 
-        return mosaic_reader(mosaic_assets, _reader, tile_x, tile_y, tile_z, **kwargs)
+        img, used_assets = mosaic_reader(
+            mosaic_assets, _reader, tile_x, tile_y, tile_z, **kwargs
+        )
+        return img, [x["id"] for x in used_assets]
```

Comment:
This was kinda a bug because previously we were returning an ImageData and a list of Items (Dict). We now make sure we return a list of Item Ids